admin/packaging.py

```diff
@@ -867,10 +875,10 @@ def omnibus_package_builder(
     return BuildSequence(
         steps=(
             InstallVirtualEnv(virtualenv=virtualenv),
-            InstallApplication(virtualenv=virtualenv,
-                               package_uri='pip==8.1.1'),
```

Comment:
The virtualenv package has been updated so we get the latest pip as part of that.